### ISSUE TITLE
Design/ Layout issue on KDE

### ISSUE THREAD

**Issue by SpaceKookie** [Valama/valama#172]
From the screenshots it's aparent that the application was designed for Unity/ GNOME in mind. On KDE it's actually quite ugly and I hope that there is something you can do about it :)

![valama](https://cloud.githubusercontent.com/assets/7669898/6139106/27d00a24-b188-11e4-8224-8afee59e70f4.png)

**Comment by inizan-yannick** [comment]
according to this french page : http://blog.rom1v.com/2012/05/configurer-le-theme-des-applications-gtk-sous-kde/ , try to install gtk3-engines-oxygen package

**Comment by SpaceKookie** [comment]
Hey there, thanks for that. Well...it looks a little better now, though GTK doesn't use the default window manager borders as I would expect it :(

![valama2](https://cloud.githubusercontent.com/assets/7669898/6139473/6609b890-b18c-11e4-884a-3ccc934a5ff5.png)

**Comment by inizan-yannick** [comment]
yes, Gtk Window with header bar is special : hide original title bar and defaults options
